firebase/fcm/공지알림.py

```
@@ -0,0 +1,208 @@
+        return messaging.send(
+            message,
+            dry_run=dry_run,
+            app=firebase_app,
+        )
+    except messaging.UnregisteredError:
+        print("❌ 만료되었거나 등록 해제된 FCM 토큰입니다.")
+    except messaging.SenderIdMismatchError:
+        print("❌ FCM 토큰과 Firebase 프로젝트가 일치하지 않습니다.")
+    except messaging.ThirdPartyAuthError:
+        print("❌ Firebase의 APNs 인증키가 유효하지 않습니다.")
+    except Exception as error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)